pkg/xds/generator/testdata/inbound-proxy/4-envoy-config.golden.yaml

```diff
@@ -122,6 +118,10 @@ resources:
                 safeRegexMatch:
                   googleRe2: {}
                   regex: .*&kuma.io/service=[^&]*frontend[,&].*
+          - name: envoy.filters.http.local_ratelimit
```

Comment:
Is this reordering expected and/or is it itself a fix for something?

Reply:
Yeh, the reordering is expected since filters will now apply in the order specified, rather than the reverse order. AFAICT this OK. We don't currently have filters where ordering is critical.